src/components/Person/Person.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Person as PersonType } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { useParams } from 'react-router-dom';
+
+interface Props {
+  person: PersonType;
+}
+export const Person: React.FC<Props> = ({ person }) => {
+  const { sex, born, died, fatherName, motherName, mother, father } = person;
+  const fName = fatherName || '-';
+  const mName = motherName || '-';
+  const { slug } = useParams();
+  const className = slug === person.slug ? 'has-background-warning' : '';
```

Comment:
The use of `useParams` to get the `slug` assumes that the URL parameter is named `slug`. Ensure that the route configuration in your application matches this assumption. Also, verify that `person.slug` exists in the `PersonType` interface.